src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable linebreak-style */
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] As already mentioned above, let's kindly follow the duct pattern. It's very important especially if you may decide to scale your project in the future.